{"Timestamp":"2020-03-17T16:10:38.7089978-03:00","Level":"Information","MessageTemplate":"Starting web host"}
{"Timestamp":"2020-03-17T16:10:39.6921332-03:00","Level":"Debug","MessageTemplate":"Attempting to connect to TCP endpoint {host} after delay of {delay} seconds","Properties":{"host":"tls://localhost:5600/","delay":1}}
{"Timestamp":"2020-03-17T16:10:43.7068633-03:00","Level":"Error","MessageTemplate":"failure inside TCP socket: {message} - socket error found {socketErrorCode}","Exception":"System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): No connection could be made because the target machine actively refused it. [::ffff:127.0.0.1]:5600\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\r\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\r\n   at System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.TcpClient.<>c.<ConnectAsync>b__28_1(IAsyncResult asyncResult)\r\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Serilog.Sinks.Network.Sinks.TCP.TcpSocketWriter.<>c__DisplayClass9_0.<<-ctor>b__0>d.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Serilog.Sinks.Network.Sinks.TCP.ExponentialBackoffTcpReconnectionPolicy.ConnectAsync(Func`2 connect, Uri host, CancellationToken cancellationToken)","Properties":{"message":"No connection could be made because the target machine actively refused it. [::ffff:127.0.0.1]:5600","socketErrorCode":"ConnectionRefused"}}
{"Timestamp":"2020-03-17T16:11:54.9781141-03:00","Level":"Information","MessageTemplate":"Starting web host"}
{"Timestamp":"2020-03-17T16:11:55.6980433-03:00","Level":"Debug","MessageTemplate":"Attempting to connect to TCP endpoint {host} after delay of {delay} seconds","Properties":{"host":"tls://localhost:5600/","delay":1}}
{"Timestamp":"2020-03-17T16:11:59.7113168-03:00","Level":"Error","MessageTemplate":"failure inside TCP socket: {message} - socket error found {socketErrorCode}","Exception":"System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): No connection could be made because the target machine actively refused it. [::ffff:127.0.0.1]:5600\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\r\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\r\n   at System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.TcpClient.<>c.<ConnectAsync>b__28_1(IAsyncResult asyncResult)\r\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Serilog.Sinks.Network.Sinks.TCP.TcpSocketWriter.<>c__DisplayClass9_0.<<-ctor>b__0>d.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Serilog.Sinks.Network.Sinks.TCP.ExponentialBackoffTcpReconnectionPolicy.ConnectAsync(Func`2 connect, Uri host, CancellationToken cancellationToken)","Properties":{"message":"No connection could be made because the target machine actively refused it. [::ffff:127.0.0.1]:5600","socketErrorCode":"ConnectionRefused"}}
{"Timestamp":"2020-03-17T19:18:49.3491914+00:00","Level":"Information","MessageTemplate":"Starting web host"}
{"Timestamp":"2020-03-17T19:18:50.5769646+00:00","Level":"Debug","MessageTemplate":"Attempting to connect to TCP endpoint {host} after delay of {delay} seconds","Properties":{"host":"tls://localhost:5600/","delay":1}}
{"Timestamp":"2020-03-17T19:18:50.5899506+00:00","Level":"Error","MessageTemplate":"failure inside TCP socket: {message} - socket error found {socketErrorCode}","Exception":"System.Net.Sockets.SocketException (99): Cannot assign requested address\n   at System.Net.Sockets.Socket.BeginConnectEx(EndPoint remoteEP, Boolean flowContext, AsyncCallback callback, Object state)\n   at System.Net.Sockets.Socket.UnsafeBeginConnect(EndPoint remoteEP, AsyncCallback callback, Object state, Boolean flowContext)\n   at System.Net.Sockets.Socket.PostOneBeginConnect(MultipleAddressConnectAsyncResult context)\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\n   at System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\n   at System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)\n   at System.Net.Sockets.TcpClient.<>c.<ConnectAsync>b__28_1(IAsyncResult asyncResult)\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\n--- End of stack trace from previous location where exception was thrown ---\n   at Serilog.Sinks.Network.Sinks.TCP.TcpSocketWriter.<>c__DisplayClass9_0.<<-ctor>b__0>d.MoveNext()\n--- End of stack trace from previous location where exception was thrown ---\n   at Serilog.Sinks.Network.Sinks.TCP.ExponentialBackoffTcpReconnectionPolicy.ConnectAsync(Func`2 connect, Uri host, CancellationToken cancellationToken)","Properties":{"message":"Cannot assign requested address","socketErrorCode":"AddressNotAvailable"}}
{"Timestamp":"2020-03-17T19:18:51.6242544+00:00","Level":"Debug","MessageTemplate":"Attempting to connect to TCP endpoint {host} after delay of {delay} seconds","Properties":{"host":"tls://localhost:5600/","delay":3}}
{"Timestamp":"2020-03-17T19:18:51.6509249+00:00","Level":"Error","MessageTemplate":"failure inside TCP socket: {message} - socket error found {socketErrorCode}","Exception":"System.Net.Sockets.SocketException (99): Cannot assign requested address\n   at System.Net.Sockets.Socket.BeginConnectEx(EndPoint remoteEP, Boolean flowContext, AsyncCallback callback, Object state)\n   at System.Net.Sockets.Socket.UnsafeBeginConnect(EndPoint remoteEP, AsyncCallback callback, Object state, Boolean flowContext)\n   at System.Net.Sockets.Socket.PostOneBeginConnect(MultipleAddressConnectAsyncResult context)\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\n   at System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\n   at System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)\n   at System.Net.Sockets.TcpClient.<>c.<ConnectAsync>b__28_1(IAsyncResult asyncResult)\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\n--- End of stack trace from previous location where exception was thrown ---\n   at Serilog.Sinks.Network.Sinks.TCP.TcpSocketWriter.<>c__DisplayClass9_0.<<-ctor>b__0>d.MoveNext()\n--- End of stack trace from previous location where exception was thrown ---\n   at Serilog.Sinks.Network.Sinks.TCP.ExponentialBackoffTcpReconnectionPolicy.ConnectAsync(Func`2 connect, Uri host, CancellationToken cancellationToken)","Properties":{"message":"Cannot assign requested address","socketErrorCode":"AddressNotAvailable"}}
{"Timestamp":"2020-03-17T19:18:54.6539149+00:00","Level":"Debug","MessageTemplate":"Attempting to connect to TCP endpoint {host} after delay of {delay} seconds","Properties":{"host":"tls://localhost:5600/","delay":7}}
{"Timestamp":"2020-03-17T19:18:54.6701664+00:00","Level":"Error","MessageTemplate":"failure inside TCP socket: {message} - socket error found {socketErrorCode}","Exception":"System.Net.Sockets.SocketException (99): Cannot assign requested address\n   at System.Net.Sockets.Socket.BeginConnectEx(EndPoint remoteEP, Boolean flowContext, AsyncCallback callback, Object state)\n   at System.Net.Sockets.Socket.UnsafeBeginConnect(EndPoint remoteEP, AsyncCallback callback, Object state, Boolean flowContext)\n   at System.Net.Sockets.Socket.PostOneBeginConnect(MultipleAddressConnectAsyncResult context)\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\n   at System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(Object result, MultipleAddressConnectAsyncResult context)\n   at System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\n   at System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)\n   at System.Net.Sockets.TcpClient.<>c.<ConnectAsync>b__28_1(IAsyncResult asyncResult)\n   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)\n--- End of stack trace from previous location where exception was thrown ---\n   at Serilog.Sinks.Network.Sinks.TCP.TcpSocketWriter.<>c__DisplayClass9_0.<<-ctor>b__0>d.MoveNext()\n--- End of stack trace from previous location where exception was thrown ---\n   at Serilog.Sinks.Network.Sinks.TCP.ExponentialBackoffTcpReconnectionPolicy.ConnectAsync(Func`2 connect, Uri host, CancellationToken cancellationToken)","Properties":{"message":"Cannot assign requested address","socketErrorCode":"AddressNotAvailable"}}
